app/helpers/elasticsearch_helper.rb

```diff
@@ -14,13 +14,14 @@ def prefix_match(model, field, prefix, condition)
     results
   end
 
-  def taxon_search(prefix)
+  def taxon_search(prefix, tax_levels = %w[species genus], label_levels = false)
```

Comment:
Shouldnt the default be to search all levels?

Reply:
Good question. I think default should be just species & genus because most of the app focuses only on species & genus. People should not be tempted to use a default that's a lot more work than necessary. I just added all levels to backfill for existing functionality in the report search box, even though it's probably overkill